include/config/battle.h

```diff
@@ -257,4 +258,18 @@
 #define B_NEW_IMPACT_PALETTE            TRUE    // If set to TRUE, it updates the basic 'hit' palette.
 #define B_NEW_SURF_PARTICLE_PALETTE     TRUE    // If set to TRUE, it updates Surf's wave palette.
 
+// Testing support.
+#if TESTING
+#ifdef B_EXPANDED_MOVE_NAMES
+#undef B_EXPANDED_MOVE_NAMES
+#endif
+#define B_EXPANDED_MOVE_NAMES TRUE
+#ifdef B_EXPANDED_ITEM_NAMES
+#undef B_EXPANDED_ITEM_NAMES
+#endif
+#define B_EXPANDED_ITEM_NAMES TRUE
+#undef POKEMON_NAME_LENGTH
+#define POKEMON_NAME_LENGTH 12
+#endif
```

Comment:
I don't think it makes much sense for this to be here. Is there another place it can be moved to?